benchbuild/projects/benchbuild/tcc.py

```diff
@@ -2,45 +2,50 @@
 
 from plumbum import local
 
-from benchbuild import project
-from benchbuild.utils import compiler, download, run, wrapping
+import benchbuild as bb
+from benchbuild.source import HTTP
 from benchbuild.utils.cmd import make, mkdir, tar
 
 
-@download.with_wget({
-    '0.9.26':
-        'http://download-mirror.savannah.gnu.org/releases/tinycc/tcc-0.9.26.tar.bz2'
-})
-class TCC(project.Project):
+class TCC(bb.Project):
+    VERSION = '0.9.26'
     NAME = 'tcc'
     DOMAIN = 'compilation'
     GROUP = 'benchbuild'
-    VERSION = '0.9.26'
-    SRC_FILE = 'tcc.tar.bz2'
+    SOURCE = [
+        HTTP(
+            remote={
+                '0.9.26':
+                    'http://download-mirror.savannah.gnu.org/releases/tinycc/tcc-0.9.26.tar.bz2'  # pylint: disable=line-too-long
```

Comment:
Codacy found an issue: [Useless suppression of 'line-too-long'](https://app.codacy.com/manual/polyjit/benchbuild_2/pullRequest?prid=5694346)